README.md

```diff
@@ -1,4 +1,4 @@
-# ProximaX Blockchain SDK for TypeScript and JavaScript
+<p align="center"><a href="https://golang.org" target="_blank" rel="noopener noreferrer"><img width="300" src="https://user-images.githubusercontent.com/29048783/72934273-4cd5e700-3d31-11ea-8d67-bcfdaf0a6404.png" alt="Golang logo"></a></p>
```

Comment:
The picture is cool, but please update the golang references in "a href..." and "img alt..."